console/src/assets/i18n/de.json

```diff
@@ -128,102 +127,103 @@
             "EMAIL": {
                 "TITLE": "Email",
-                "VALID": "validiert",
-                "RESEND": "Verifikationsmail versenden"
+                "VALID": "Validiert",
+                "RESEND": "Verifikationsmail erneut senden"
             },
             "PHONE": {
                 "TITLE": "Telefon",
-                "VALID": "validiert",
-                "RESEND": "Verifikationsnachricht versenden"
+                "VALID": "Validiert",
+                "RESEND": "Verifikationsnachricht erneute senden"
             },
-            "RESENDCODE": "Code senden",
+            "RESENDCODE": "Code erneut senden",
             "ENTERCODE":"Verifizieren",
             "ENTERCODE_DESC":"Code verifizieren"
         },
-        "ADDRESS": {
-            "TITLE": "Adresse",
-            "STREET": "Strasse/Nr",
-            "LOCALITY": "Ort",
-            "POSTAL_CODE": "Postleitzahl",
-            "REGION": "Kanton",
-            "COUNTRY": "Land"
+        "GRANTS": {
+            "TITLE": "Benutzer Grants",
+            "DESCRIPTION": "Erteilen Sie diesem Benutzer Verwaltunszugriff auf bestimmte Projekte",
+            "CREATE": {
+                "TITLE": "Benutzer Grant erstellen",
+                "DESCRIPTION": "Suchen Sie nach der Organisation, dem Projekt und den verfügbaren Rollen!"
+            }
         },
         "VALIDATION": {
-            "INVALIDPATTERN": "Das Password muss aus mindestens 8 Zeichen bestehen und einen Grossbuchstaben, ein Sonderzeichen und eine Zahl enthalten. Die maximale Anzahl an Zeichen ist 72!",
-            "REQUIRED": "Das Feld ist leer",
+            "INVALIDPATTERN": "## TODO - Replace with Password Policy text - TODO ##",
+            "REQUIRED": "Das Eingagefeld ist leer",
             "MINLENGTH":"Das Password muss mindestens {{requiredLength}} Zeichen lang sein!",
-            "NOEMAIL":"Username darf keine email sein!",
+            "NOEMAIL":"Username darf keine Email Adresse sein!",
             "UPPERCASEMISSING":"Password muss einen Großbuchstaben beinhalten",
             "LOWERCASEMISSING":"Password muss einen Kleinbuchstaben beinhalten",
             "SYMBOLERROR":"Das Password muss ein Symbol beinhalten!",
             "NUMBERERROR":"Das Password muss eine Nummer beinhalten!"
         },
         "STATE": {
-            "0":"unbekannt",
-            "1":"aktiv",
-            "2":"abgelaufen"
+            "0":"Unbekannt",
+            "1":"Aktiv",
+            "2":"Abgelaufen"
         },
         "SEARCH":{
-            "FOUND":"gefunden"
+            "FOUND":"Gefunden"
         },
         "TARGET": {
-            "SELF":"Suchen Sie oben nach Usern ihrer Organisation, Zum hinzufügen externer Benutzer",
-            "EXTERNAL":"Um User Ihrer Organisation hinzuzufügen",
+            "SELF":"Suchen Sie nach den Benutzern ihrer Organisation, Zum hinzufügen externer Benutzer", #DAFUQ
+            "EXTERNAL":"Um User Ihrer Organisation hinzuzufügen", #DAFUQ
```

Comment:
@fgerschwiler @peintnermax i have no clue what this does 😁

Reply:
```suggestion
            "SELF":"Um einen Benutzer aus einer anderen Organisation zu berechtigen",
            "EXTERNAL": "Um einen Benutzer ihrer eigenen Organisation zu berechtigen", 
``